From 85c9ab4e58717f1f415c6f5cba85826228341b7c Mon Sep 17 00:00:00 2001
From: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
Date: Tue, 26 Sep 2023 15:00:34 +0530
Subject: [PATCH] spi: spidev: add compatible string "qti,spidev"

Since spidev can no more be referenced in DT without a specific
compatible string, adding compatible string "qti,spidev".

Signed-off-by: Akila N <akilan@codeaurora.org>
(cherry picked from commit 58e3594718737e8b9d240ba04970784c3ce8a945)

Change-Id: I328f720dc4c60f107cca422e982afd7825efa040
Signed-off-by: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index 485c5a96367f..f99ff5827597 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -727,6 +727,7 @@ static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "cisco,spi-petra", .data = &spidev_of_check },
 	{ .compatible = "micron,spi-authenta", .data = &spidev_of_check },
 	{ .compatible = "siliconlabs,si3210" },
+	{ .compatible = "qti,spidev" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
2.34.1

